codegen-renderers/src/main/kotlin/io/vrap/codegen/languages/php/extensions/EObjectTypeExtensions.kt

```diff
@@ -1,9 +1,10 @@
 package io.vrap.codegen.languages.php.extensions
 
 import io.vrap.codegen.languages.ExtensionsBase
+import io.vrap.rmf.codegen.types.VrapNilType
 import org.eclipse.emf.ecore.EObject
 
-interface EObjectTypeExtensions : io.vrap.codegen.languages.ExtensionsBase {
-    fun EObject.toVrapType() = vrapTypeProvider.doSwitch(this)
+interface EObjectTypeExtensions : ExtensionsBase {
+    fun EObject?.toVrapType() = if(this !=null ) vrapTypeProvider.doSwitch(this) else VrapNilType()
```

Comment:
This looks a bit weird, but somehow fixes an issue? Would be great if you could explain the problem here a bit, may be there's even a bug in RMF?

Reply:
yep, actually there was a case some method had a return but the return type was null, so add to this to handle null cases.